File: app/src/main/java/com/sdei/parentIn/model/BaseModel.java
package com.sdei.parentIn.model;

import com.sdei.parentIn.utils.ConstantsKt;

public class BaseModel {


    public BaseModel() {
        this.statusCode = ConstantsKt.getCODE_ERROR();
        this.message ="";
    }

    /**
     * statusCode : 200
     * message : Registration successful
     */

    private int statusCode;
    private String message;

    public int getStatusCode() {
        return statusCode;
    }

    public void setStatusCode(int statusCode) {
        this.statusCode = statusCode;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: app/src/main/java/com/sdei/parentIn/model/BaseModel.kt
package com.sdei.parentIn.model

import com.sdei.parentIn.interfaces.InterConst.CODE_ERROR

open class BaseModel {


    /**
     * statusCode : 200
     * message : Registration successful
     */

    var statusCode = CODE_ERROR
    var message: String = ""

    constructor() {
        this.statusCode = CODE_ERROR
        this.message = ""
    }

    constructor(message: String) {
        this.message = message
    }

    constructor(statusCode: Int, message: String){
        this.message = message
        this.statusCode = statusCode
    }
}
